sdk/core/src/evm/storageHubClient.ts

```diff
@@ -36,10 +36,11 @@ type FileSystemContract<TClient extends EvmClient> = GetContractReturnType<
 >;
 
 /**
- * Internal constant precompile address for FileSystem on StorageHub runtimes.
+ * Default precompile address for FileSystem on StorageHub runtimes.
  * If a chain uses a different address, this constant should be updated accordingly.
  */
-const FILE_SYSTEM_PRECOMPILE_ADDRESS = "0x0000000000000000000000000000000000000064" as Address;
+export const FILE_SYSTEM_PRECOMPILE_ADDRESS =
```

Comment:
```suggestion
export const SH_FILE_SYSTEM_PRECOMPILE_ADDRESS =
```
Just to make it EXTRA clear that's the StorageHub address